src/main/java/org/interledger/InterledgerAddress.java

```diff
@@ -29,7 +30,27 @@
 public interface InterledgerAddress {
 
   /**
-   * Return this address's value as a non-null {@link String}.  For example:
+   * Constructor to allow quick create of String.
+   *
+   * @param value String representation of an Interledger Address
+   */
+  static InterledgerAddress of(final String value) {
```

Comment:
Its such a habit for me to do it, i forgot about the implicit public modifier. Here's the official word from the [spec](http://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html)

> Every method declaration in the body of an interface is implicitly public (§6.6). It is permitted, but discouraged as a matter of style, to redundantly specify the public modifier for a method declaration in an interface.

I'll keep that one in mind :)